src/components/commonPagination.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="common-pagination">
     <el-pagination
-      v-if="meta.count && meta.count > defaultPageSizeOpt[0]"
+      v-if="
+        meta.count &&
+        meta.count > (meta.limit >= defaultPageSizeOpt[0] ? defaultPageSizeOpt[0] : meta.limit)
```

Comment:
This expression seems a bit complex; perhaps we could convert it into an easier-to-understand function.